ui/components/src/InfoTooltip/InfoTooltip.tsx

```diff
@@ -33,15 +33,27 @@ interface InfoTooltipProps {
   id?: string;
   title?: string;
   placement?: TooltipPlacement;
+  enterDelay?: number; // default is 500ms
+  enterNextDelay?: number; // default is 500ms
 }
 
-export const InfoTooltip = ({ id, title, description, placement, children }: InfoTooltipProps) => {
+export const InfoTooltip = ({
+  id,
+  title,
+  description,
+  placement,
+  children,
+  enterDelay,
+  enterNextDelay,
+}: InfoTooltipProps) => {
   return (
     <StyledTooltip
       arrow
       id={id}
-      placement={placement}
+      placement={placement ?? TooltipPlacement.Top}
       title={<TooltipContent title={title} description={description} />}
+      enterDelay={enterDelay ?? 500}
```

Comment:
Curious, why are these delays needed?

Reply:
once you learn what the icons mean, the tooltips will be very distracting if they show up every time.

discussed with @foxbat07 and agreed that 500ms seems acceptable